packages/browser-pool/src/index.ts

```diff
@@ -54,3 +54,9 @@ export type {
     GetFingerprintReturn,
 } from './fingerprinting/types';
 export type { InferBrowserPluginArray } from './utils';
+export type {
+    PuppeteerController,
+} from './puppeteer/puppeteer-controller';
+export type {
+    PlaywrightController,
+} from './playwright/playwright-controller';
```

Comment:
this will conflict with the classes in pw and pup crawler packages, or not?

Reply:
We don't have any class called Puppeteer/PlaywrightController in them (and the build tests pass) 🤷‍♂️

Reply:
Oh thooose. Yeah good catch, can delete those now 🎉